vnext/Desktop.DLL/React.Windows.Desktop.DLL.vcxproj

```diff
@@ -23,12 +23,13 @@
     <ProjectGuid>{88BAB0FA-E1AC-4DA7-A30C-F91702A8EADB}</ProjectGuid>
     <ProjectName>React.Windows.Desktop.DLL</ProjectName>
     <TargetName>react-native-win32</TargetName>
-    <USE_V8>true</USE_V8>
   </PropertyGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
   <Import Project="$(ReactNativeWindowsDir)PropertySheets\React.Cpp.props" />
   <PropertyGroup Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <!-- This property needs to be overwritten after the inclusion of React.Cpp.props -->
```

Comment:
I don't think this is right:
The way I wrote this is that you set USE_V8 to true, and then React.Cpp.props would only set it to false if it is empty or if we are building ARM or ARM64 - since there is no v8 for those archs

Reply:
msbuild is weird, it didn't work that way. You can confirm by trying to build ReactWindows-Desktop.sln and observing there is no v8jsi.dll in the target folder, and the JSI.Desktop.UnitTests don't execute the V8 tests (19 tests instead of 35).

---
In reply to: [422401551](https://github.com/microsoft/react-native-windows/pull/4840#discussion_r422401551) [](ancestors = 422401551)

Reply:
can you clarify what you mean by "msbuild is weird"? i.e. what is causing the behavior you describe?

Reply:
Ah, I think it makes sense. This was a logic error in the conditional. Let me try turning everything around.

---
In reply to: [422403167](https://github.com/microsoft/react-native-windows/pull/4840#discussion_r422403167) [](ancestors = 422403167,422401551)